tools/universe/package_builder.py

```diff
@@ -15,7 +15,7 @@
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.DEBUG, format="%(message)s")
 
-_jre_url = 'https://downloads.mesosphere.com/java/jre-8u144-linux-x64.tar.gz'
+_jre_url = 'https://downloads.mesosphere.io/java/jre-8u152-linux-x64.tar.gz'
```

Comment:
Thanks @vespian. We already have #1866 open to track this. Any reason for the `.io` to `.com` change as well?

Reply:
>  We already have #1866 open to track this
OK, then I am closing this one
> Any reason for the .io to .com change as well?
None, I just picked the first url that appeared in the Jenkins job log [1]. After reading your comment, I verified that both `.com` and `.io` serve the same content. Should we stick to `.com`/is there a policy/recommendation for that?

[1] https://jenkins.mesosphere.com/service/jenkins/view/Infinity/job/infinity-tools/job/upload-production-asset/